tests/lit.cfg.py

```diff
@@ -20,7 +20,7 @@
     os.makedirs(unittest_build_path)
 data_path = os.path.join(config.halo_build_dir, 'tests/unittests/data')
 if not os.path.exists(data_path):
-    os.system('git clone -b rel-1.5.0 https://github.com/onnx/onnx.git')
+    os.system('git clone -b rel-1.8.0 https://github.com/onnx/onnx.git')
```

Comment:
add "--depth=1" ? (same for the other script that clones it).

btw, why do we have the "clone" in two files?

Reply:
in run_test.py, it uses to generate test cases[some cc files] , here it downloads the test data[input/output data]